src/argilla/server/policies.py

```diff
@@ -117,6 +117,12 @@ async def is_allowed(actor: User) -> bool:
         return is_allowed
 
 
+class UserPolicyV1:
+    @classmethod
+    async def list_workspaces(cls, actor: User) -> bool:
+        return actor.is_owner
```

Comment:
I think we should list workspaces to users requesting their own workspaces, I mean, when actor.id == user_id
```suggestion
    async def list_workspaces(cls, user_id:UUID) -> PolicyAction:
        return lambda actor: actor.is_owner or actor.id == user_id
```